### PR TITLE
Add active connections list tab to the webUI

### PR DESCRIPTION
**Pull request by sfc-gh-pkommini**
Changes
- Add connections data to `src/webui/backend/webui/api/data.py`
- Pass it to the UI
- Add `Active Connections` tab on the connectors page
- Additionally fix the extra double quotes on the outside of serialized timestamps

Testing
Tested the UI on Oz and got the following connections:
image

### CODE REVIEW

File: src/webui/frontend/src/routes/Dashboard/Connectors.tsx [hunk 1]
```diff
@@ -146,36 +146,36 @@ class Connectors extends React.Component<ConnectorsProps & {path: string}, OwnSt
...
-                          this.changeOption(opt.name, opt.default);
-                        }
-                      },
-                      onChange: (e: any) => {
-                        // todo why doesn't ref to e work here w/ prevState?
-                        this.changeOption(opt.name, e.target.value);
-                      },
-                    })
-                  )}
+                      React.createElement(opt.secret || opt.mask_on_screen ? Input.Password : Input, {
```
Comment (sfc-gh-afedorov) [inline src/webui/frontend/src/routes/Dashboard/Connectors.tsx:149]: not sure whether this is text editor or accident but let's keep consistent indent, i.e. 2-space indent in this file

File: src/webui/frontend/src/routes/Dashboard/Connectors.tsx [hunk 2]
```diff
@@ -213,28 +213,61 @@ class Connectors extends React.Component<ConnectorsProps & {path: string}, OwnSt
...
-              avatar={<Avatar src={`/icons/connectors/${c.name}.png`} />}
-              title={c.title}
-              description={c.description}
-              style={{height: 75}}
-            />
-          </Card>
-        ))}
-      </BasicLayout>
-    );
+        <BasicLayout>
```
Comment (sfc-gh-afedorov) [inline src/webui/frontend/src/routes/Dashboard/Connectors.tsx:216]: ditto here -- 2 space indent, please

File: src/webui/frontend/src/store/data.ts [hunk 3]
```diff
@@ -41,3 +41,19 @@ export class Baseline {
...
+  row_count: number;
+
+  constructor(cti: ConnectionPayload) {
+    this.raw = cti;
+    this.table_name = cti.name;
+    this.created_on = cti.created_on;
+    this.byte_count = cti.bytes;
+    this.row_count = cti.rows;
+  }
+}
```
Comment (sfc-gh-afedorov) [inline src/webui/frontend/src/store/data.ts:59]: can you set your editor to add newline at end of file? https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

**Review by sfc-gh-afedorov — APPROVED** [review]
lgtm overall, please fix the comments and double check that the "created_on" date is something nicer than ISO8601 and we can add that detail on mouse-hover either via native tooltip or https://ant.design/components/tooltip/?

File: src/webui/frontend/src/routes/Dashboard/Connectors.tsx [hunk 4]
```diff
@@ -213,28 +213,61 @@ class Connectors extends React.Component<ConnectorsProps & {path: string}, OwnSt
...
+                pagination={false}
+                columns={[{
+                  title: 'Name',
+                  dataIndex: 'table_name',
+                  key: 'name'
+                },
+                {
+                  title: 'Created On',
+                  dataIndex: 'created_on',
+                  key: 'created_on'
```
Comment (sfc-gh-afedorov) [inline src/webui/frontend/src/routes/Dashboard/Connectors.tsx:253]: let's add a `render` here to format the date

```
c => (new Date(c)).toLocaleDateString()
> "10/16/2020"
```

File: src/webui/frontend/src/routes/Dashboard/Connectors.tsx [hunk 5]
```diff
@@ -213,28 +213,61 @@ class Connectors extends React.Component<ConnectorsProps & {path: string}, OwnSt
...
+                },
+                {
+                  title: 'Byte Count',
+                  dataIndex: 'byte_count',
+                  key: 'byte_count'
```
Comment (sfc-gh-afedorov) [inline src/webui/frontend/src/routes/Dashboard/Connectors.tsx:258]: ditto, but maybe with

```
(bytes) => {
    if (bytes === 0) return '0 Bytes';

    const k = 1024;
    const dm = decimals < 0 ? 0 : decimals;
    const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];

    const i = Math.floor(Math.log(bytes) / Math.log(k));

    return parseFloat((bytes / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i];
}
```

?

File: src/webui/frontend/src/routes/Dashboard/Connectors.tsx (hunk 2 above)
Comment (sfc-gh-pkommini) [reply src/webui/frontend/src/routes/Dashboard/Connectors.tsx:216]: Some autoformat magic happening here. Sorry about that. Will fix now.

File: src/webui/frontend/src/routes/Dashboard/Connectors.tsx (hunk 1 above)
Comment (sfc-gh-pkommini) [reply src/webui/frontend/src/routes/Dashboard/Connectors.tsx:149]: The default autoformatter seems to be using a different style guide. I switched the formatter and now it seems to do 2-space indent for that particular construct.

File: src/webui/frontend/src/store/data.ts (hunk 3 above)
Comment (sfc-gh-pkommini) [reply src/webui/frontend/src/store/data.ts:59]: Done.

File: src/webui/frontend/src/routes/Dashboard/Connectors.tsx (hunk 5 above)
Comment (sfc-gh-pkommini) [reply src/webui/frontend/src/routes/Dashboard/Connectors.tsx]: Good feedback. Results are good.

image

File: src/webui/frontend/src/routes/Dashboard/Connectors.tsx (hunk 4 above)
Comment (sfc-gh-pkommini) [reply src/webui/frontend/src/routes/Dashboard/Connectors.tsx]: Good feedback.

**Pull request merged**